TAS.Avalonia/Communication/CommunicationWrapper.cs

```diff
@@ -0,0 +1,5 @@
+namespace TAS.Avalonia.Communication;
+
+public static class CommunicationWrapper {
+    public static string ReturnData;
```

Comment:
`ReturnData` can be a private member of `StudioCommunicationServer` rather than introducing a static class.

```cs
private string _returnData;
```